subsys/logging/Kconfig

```diff
@@ -492,6 +492,12 @@ config LOG_BACKEND_NET_MAX_BUF_SIZE
 	  IPv6 the size is 1180 octets. As each buffer will use RAM, the value
 	  should be selected so that typical messages will fit the buffer.
 
+config LOG_BACKEND_NET_SYST_ENABLE
```

Comment:
I was not marked as a reviewer for this PR so did not had a chance to comment this. I think this was merged a bit prematurely as now if user enables `CONFIG_LOG_BACKEND_NET` and `CONFIG_LOG_BACKEND_NET_SYST_ENABLE`, the loggers net backend prepares to send syslog formatted messages out but this will fail unless user sets the syslog server hostname. The code is prepared to handle that issue but there is now just useless code in the system if user only meant to activate syst logging and not syslog one.
Better would have been to have separate option for enabling syslog formatted network logging, just like the syst one.